pkg/kor/exporter.go

```diff
@@ -0,0 +1,73 @@
+package kor
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"k8s.io/client-go/kubernetes"
+)
+
+var (
+	orphanedResourcesCounter = prometheus.NewCounterVec(
```

Comment:
I think the correct type here would be a gauge(since we are not counting), and set the value to 1.
WDYT?

Reply:
yea it should be either gauge or SetToCurrentTime()